homeassistant/components/lcn/__init__.py

```diff
@@ -136,7 +138,11 @@ async def async_setup_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> b
     for service_name, service in SERVICES:
         if not hass.services.has_service(DOMAIN, service_name):
             hass.services.async_register(
```

Comment:
Ideally we migrate the services to be set up in `async_setup`. This way they don't depend on a working config entry. (but this is not for this PR)

Reply:
Ok, thanks for the info. I'll fix this in a separate PR.

Reply:
I addressed this issue in #130603